doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -307,6 +307,8 @@ Bluetooth samples
 
   * Split the configuration of the :ref:`mod_dm` module from the :ref:`nrf_dm`.
     This allows the use of the Nordic Distance Measurement library without the module.
+  * Added support for the nRF5340 target.
+  * Added support for the high accuracy distance estimate, which uses more compute-intensive algorithms to generate a distance estimate.
```

Comment:
In the header files, you use high precision. Which ever you decide to use, it requires a hyphen -> high-accuracy or high-precision.

Reply:
By the way, they seem to be a bit different things: https://www.skyhook.com/blog/advertising/the-difference-between-location-accuracy-and-precision-and-why-you-need-to-know.

Anyway, a hyphen is needed here, because it is used as an attribute to the distance estimate.

Reply:
Good point, corrected.